src/model.py

```diff
@@ -25,7 +25,69 @@ def classify_frames(self):
         # use multiprocessing on loop in list comprehension below
         return [Worker.classify_img(frame) for frame in frames]
         '''
-        pass
+        return [(6.66, 0.6)]
+
+    def has_valid_args_interpret_results(self, results, cutoff):
+
+        if not isinstance(results, type([])):
+            raise TypeError("Expected List. Got {}".format(type(results)))
+            return -1
+        timeSet = set()
+        for elt in results:
+            if elt[0] < 0:
+                raise ValueError("Negative time stamp in results. Please don't do that. Got {}".format(elt[0]))
+                return -1
```

Comment:
Aren't the return values made superfluous by the exceptions?

Reply:
Yup! Getting rid of them now.